src/Bookings.js

```diff
@@ -1,18 +1,40 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+
+const Bookings = ({ newBooking }) => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => {
+        setBookings(data);
+        setLoading(true);
```

Comment:
Here you are effectively saying that the `loading` state is `true` _after_ the bookings have been fetched from the API. It should be that the component has stopped `loading` once the `fetch` has been successful - you may need to think about how the `loading` is intialised